### PR TITLE
ZNTA-2786 fixed links in admin dashboard

### PR DESCRIPTION
**Pull request by ghost**
JIRA issue URL: https://zanata.atlassian.net/browse/ZNTA-2786

fixed the dead links in admin dashboard. 

## Checklist

- JIRA link
- Check target branch
- Make sure all commit statuses are green or otherwise documented reasons to ignore
- QA needs to evaluate against the JIRA ticket
- Changed files and commits make sense for this PR

See [Zanata Development Guidelines](https://github.com/zanata/zanata-platform/wiki/Development-Guidelines) more for information.

----
*This template can be updated in .github/PULL_REQUEST_TEMPLATE.md*

### CODE REVIEW

**Review by efloden — APPROVED** [review]

File: server/zanata-frontend/src/app/containers/Admin/index.js [hunk 1]
```diff
@@ -92,31 +92,31 @@ class Admin extends Component {
...
+                  link='rolerules' useHref>
                   <Icon name='assign' className='s2' />
                   Role assignment rules
                 </Link>
               </Col>
               <Col xs={24} sm={12} lg={6}>
                 <Link id='Translation_Memory_home'
                   className='list-group-item'
-                  link='/tm' useHref>
+                  link='/zanata/tm' useHref>
```
Comment (derkoe) [inline server/zanata-frontend/src/app/containers/Admin/index.js:103]: guess this won't work when not running under '/zanata'

**Comment by derkoe** [comment]
Created another PR #1048 that fixes even more links and the context-path issue mentioned above

File: server/zanata-frontend/src/app/containers/Admin/index.js (hunk 1 above)
Comment (ghost) [reply server/zanata-frontend/src/app/containers/Admin/index.js:103]: Thank you. I will close PR. Lets use yours.

**Comment by ghost** [comment]
Closed related to: https://github.com/zanata/zanata-platform/pull/1048

**Pull request closed without merging**